test/utils/runners.go

```diff
@@ -1194,6 +1194,10 @@ func CreatePodWithPersistentVolume(ctx context.Context, client clientset.Interfa
 		// PVs are cluster-wide resources.
 		// Prepend a namespace to make the name globally unique.
 		pv.Name = fmt.Sprintf("%s-%s", namespace, pv.Name)
+		pvs := pv.Spec.PersistentVolumeSource
+		if pvs.CSI != nil {
+			pvs.CSI.VolumeHandle = pv.Name
+		}
```

Comment:
I might have mentioned it before, but scheduler_perf seems to be the only user of these runners. You might want to move them into a directory owned by SIG Scheduling.

But not in this PR....

/approve